vsts-api-patcher/src/main.rs

```diff
@@ -1918,6 +1918,14 @@ impl Patcher {
                 self.walker(&new_key, v);
             }
         }
+        if let JsonValue::Array(_) = value {
+            for (i, v) in value.members_mut().enumerate() {
+                let mut new_key = key.to_owned();
+                let i = i.to_string();
+                new_key.push(i.as_str());
```

Comment:
it's not ideal the index being the integer, perhaps we should have a special value for indices so patch functions can use the key to match on any object in an array (to do that now would be fiddly).

conversely, in the jobject patch function I don't use the key, so this may not need solving immediately.

Reply:
Yeah, I agree this is good enough for now.